client/src/ui/Profile.tsx

```diff
@@ -1,48 +1,99 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import ProfileCard from "./ProfileCard";
-import Navbar from './Navbar';
+import "./css/ClassView.css";
+import "./css/App.css";
+import "rodal/lib/rodal.css";
+import "./css/Form.css";
+import "./css/ResultsDisplay.css";
+import { Review as ReviewType } from "common";
 
 import styles from "./css/Profile.module.css";
+import CourseReviews from "./CourseReviews";
+import axios from "axios";
+
+type netId = string;
 
 type ProfileProps = {
   imageSrc: any;
-  verifiedEmail: string;
+  netId: string;
 };
 
 export default function Profile({
   imageSrc = "/profile_bear.png",
-  verifiedEmail = "myl39@cornell.edu",
+  //for LOCAL TESTING use axl4 for no reviews, ag974 for past reviews, sj598 for pending + past reviews
+  netId = "sj598",
```

Comment:
It might be a good idea to take the default value out before merging. Not sure it would change anything though.

Reply:
I think this will be handled when the user authentication is linked!